src/components/buttons/button-set.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import useStyles from './button-set.styles';
+
+type ButtonSetProps = {
+  children?: React.ReactChild | React.ReactChild[],
```

Comment:
I think this is the official type for this...
```suggestion
  children?: React.ReactNode,
```